<launch>
  <node name="lidar_publisher"  pkg="lidar_ros_driver"  type="lidar_ros_driver_node" output="screen" respawn="false" >
    <!-- string property -->
    <param name="port"         type="string" value="192.168.0.11"/>  
    <param name="frame_id"     type="string" value="laser_frame"/>
    <param name="baudrate"         type="int" value="8090"/>  

    <!-- 0:TYPE_LIDAR -->
    <param name="lidar_type"       type="int" value="0"/>  

    <!-- float property -->
    <param name="angle_min"    type="double" value="-150.0" />
    <param name="angle_max"    type="double" value="150.0" />
    <param name="range_min"    type="double" value="0.1" />
    <param name="range_max"    type="double" value="30.0" />
    <param name="frequency"    type="double" value="20.0"/>
    <param name="sample_rate"         type="int" value="40"/>
  </node>

  <node pkg="tf" type="static_transform_publisher" name="base_link_to_laser4"
    args="0.0 0.0 0.2 0.0 0.0 0.0 /base_footprint /laser_frame 40" />
    
</launch>
